domains/selvakumaran.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "selvaklnc",
+        "email": "selvakumaran301@gmail.com"
+    },
+    "record": {
+        "CNAME": "selvaklnc.github.io"
```

Comment:
Invalid website... Website does not exist. (404)

Reply:
Hi sir, I've mentioned CNAME as "selvaklnc.github.io" and the site is at https://selvaklnc.github.io/sk and it is configured to selvakumaran.is-a.dev
Shoul I change the CNAME to selvaklnc.github.io/sk ?

Reply:
Oh, since you have a website on that domain it should be fine, my bad I couldn't find the repo on your profile either because it is named sk...
You can access the website at `selvakumaran.is-a.dev/sk`
And no, since CNAME records can't contain slash